spec/controllers/account_controller_spec.rb

```diff
@@ -312,11 +310,11 @@
           is_expected.to respond_with :redirect
           expect(assigns[:user]).not_to be_nil
           is_expected.to redirect_to(my_first_login_path)
-          expect(User.last(conditions: { login: 'register' })).not_to be_nil
+          expect(User.where(login: 'register').last).not_to be_nil
         end
 
         it 'set the user status to active' do
-          user = User.last(conditions: { login: 'register' })
+          user = User.where(login: 'register').last
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.